common/src/main/java/earth/terrarium/hermes/utils/ElementParsingUtils.java

```diff
@@ -100,4 +102,15 @@ public static boolean parseBoolean(Map<String, String> parameters, String key, b
         return defaultValue;
     }
 
+    public static Alignment parseAlignment(Map<String, String> parameters, String key, Alignment defaultValue) {
+        if (parameters.containsKey(key)) {
+            try {
+                return Alignment.fromString(parameters.get(key).toUpperCase(Locale.ROOT));
```

Comment:
move the toUpperCase to inside the fromString method itself

Reply:
Done